cmd/opentelemetry-collector/app/receiver/jaegerreceiver/jaeger_receiver.go

```diff
@@ -48,9 +51,58 @@ func (f *Factory) Type() configmodels.Type {
 func (f *Factory) CreateDefaultConfig() configmodels.Receiver {
 	cfg := f.Wrapped.CreateDefaultConfig().(*jaegerreceiver.Config)
 	cfg.RemoteSampling = createDefaultSamplingConfig(f.Viper)
+	configureAgent(f.Viper, cfg)
+	configureCollector(f.Viper, cfg)
 	return cfg
 }
 
+func configureAgent(v *viper.Viper, cfg *jaegerreceiver.Config) {
+	aOpts := agentApp.Builder{}
+	aOpts.InitFromViper(v)
+	if v.IsSet(thriftBinaryHostPort) {
+		cfg.Protocols["thrift_binary"] = &receiver.SecureReceiverSettings{
+			ReceiverSettings: configmodels.ReceiverSettings{
+				// TODO OTEL does not expose number of workers and queue length
+				Endpoint: v.GetString(thriftBinaryHostPort),
+			},
+		}
+	}
+	if v.IsSet(thriftCompactHostPort) {
+		cfg.Protocols["thrift_compact"] = &receiver.SecureReceiverSettings{
+			ReceiverSettings: configmodels.ReceiverSettings{
+				// TODO OTEL does not expose number of workers and queue length
+				Endpoint: v.GetString(thriftCompactHostPort),
+			},
+		}
+	}
+}
+
+func configureCollector(v *viper.Viper, cfg *jaegerreceiver.Config) {
+	cOpts := collectorApp.CollectorOptions{}
+	cOpts.InitFromViper(v)
+	if v.IsSet(collectorApp.CollectorGRPCHostPort) {
+		cfg.Protocols["grpc"] = &receiver.SecureReceiverSettings{
+			ReceiverSettings: configmodels.ReceiverSettings{
+				Endpoint: cOpts.CollectorGRPCHostPort,
+			},
+		}
+		if cOpts.TLS.CertPath != "" && cOpts.TLS.KeyPath != "" {
+			cfg.Protocols["grpc"].TLSCredentials = &receiver.TLSCredentials{
+				// TODO client-ca is missing in OTEL
```

Comment:
Has an issue been created to track this?

Reply:
yes https://github.com/open-telemetry/opentelemetry-collector/issues/963